source/documentation/monitoring-an-app/how-to-create-alarms.html.md.erb

```diff
@@ -18,14 +18,19 @@ This guide assumes the following:
 * You have [created a namespace for your application][env-create]
 
 ### Creating a slack webhook and amend Alertmanager
-This step requires the Cloud Platform team to create a receiver in [Alertmanager](https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cloud-platform-components/templates/prometheus-operator.yaml.tpl###L115) and a [Slack webhook](https://api.slack.com/incoming-webhooks).
+Create a [slack webhook](https://slack.com/intl/en-gb/help/articles/115005265063-Incoming-webhooks-for-Slack) if you dont already have one for the channel you want to send the alerts.
+Provide the webhook url securely to the Cloud Platform team to perform the next step.
```

Comment:
@digitalronin , There is a link in user guide on how to create a secret for AWS credentials and I have added for reference. 
I have updated to send the secret name as part of the PR so it is all in one place rather than in ask channel. Hope this make sense.